### ISSUE TITLE
Verse Labels Not Displaying in HTML

### ISSUE THREAD

**Issue by kralphs** [martijnversluis/ChordSheetJS#747]
Looking at the template for the HTML, there is currently nothing that handles writing Verse/Chorus annotations. I've confirmed the parser is parsing the tags correctly, but the `Song` class drops the leading `Tag` line when calculating `bodyParagraphs` so the information isn't even available for the template to format the output.

Currently I'm using a mix of Regex and CSS to insert single line comment directives as a work around, but this would seem to be an important feature, especially since the readme indicates support for these directives.

**Comment by martijnversluis** [comment]
Hey @kralphs. Thanks for reaching out, that sure sounds like a bug.

Could you perhaps post (an example of) the chord sheet that is not rendered correctly?

**Comment by kralphs** [comment]
![image](https://user-images.githubusercontent.com/46707171/203157619-32215ef2-8115-4290-a259-01637bcb9ec1.png)

As below, you can see that the first line of the only body paragraph is a chord lyric pair corresponding to the first line of text, but the last line is the `end_of_verse` tag. As I said before, the `lines` property holds all the parsed lines correctly including capturing the `start_of_verse` tag with the correct value.

![image](https://user-images.githubusercontent.com/46707171/203158155-1024bcf2-d37e-48c8-919c-8fbb4e7c4d4a.png)

**Comment by martijnversluis** [comment]
Ah wow, you actually taught me some new Chordpro knowledge ✨ ; never knew those kind of tags could actually contain a label that should be rendered.

Fixing this should be doable! I'll put this on the board and try to give this some love this week.

BTW, as an advice: if you need to override the HTML output I would recommend, if possible, to create your own formatter based on an existing one:

```js
function myTemplate({
  configuration,
  song,
  renderBlankLines = false,
  song: {
    title,
    subtitle,
    bodyParagraphs,
    metadata,
  },
}) {
  return stripHTML(`
    <div class="chord-sheet">
      ...
    </div>
  `);
}

class MyHtmlFormatter extends HtmlDivFormatter {
  get template() {
    return myTemplate;
  }
}
```

**Comment by martijnversluis** [comment]
Hey @kralphs 

I just published [`7.10.0`](https://github.com/martijnversluis/ChordSheetJS/releases/tag/v7.10.0), this should fix your issue. Could you let me know if it works for you?

**Comment by kralphs** [comment]
LGTM. As implemented, I can massage it so that the `{chorus}` directive is effectively available as well. Pretty sure this gives me all I need for my chord charts, but will def get back to you if there's other stuff I see :)
